pkg/cache/internal/deleg_map.go

```diff
@@ -68,6 +68,10 @@ func (m *InformersMap) WaitForCacheSync(stop <-chan struct{}) bool {
 	syncedFuncs := append([]cache.InformerSynced(nil), m.structured.HasSyncedFuncs()...)
 	syncedFuncs = append(syncedFuncs, m.unstructured.HasSyncedFuncs()...)
 
+	// TODO(jlanford): investigate why this sleep is necessary
+	//   to keep the tests passing. This is related to
+	//   kubernetes/client-go@6b59ea5
+	time.Sleep(time.Millisecond * 100)
```

Comment:
This code can be removed when #755 is merged.

Reply:
We def should not merge with this in place

Reply:
@DirectXMan12 this has been removed now that #755 is merged.